ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoice.java

```diff
@@ -552,18 +569,19 @@ private BlockImportResult importBlockAndState(
           payloadResult.getFailureCause().orElseThrow());
     }
 
-    LOG.debug("blobSidecars validation result: {}", blobSidecarsAndValidationResult::toLogString);
-
-    switch (blobSidecarsAndValidationResult.getValidationResult()) {
+    switch (dataAndValidationResult.validationResult()) {
+      case VALID, NOT_REQUIRED ->
+          LOG.debug("sidecars validation result: {}", dataAndValidationResult::toLogString);
       case NOT_AVAILABLE -> {
+        LOG.debug("sidecars validation result: {}", dataAndValidationResult::toLogString);
         return BlockImportResult.failedDataAvailabilityCheckNotAvailable(
-            blobSidecarsAndValidationResult.getCause());
+            dataAndValidationResult.cause());
       }
       case INVALID -> {
-        debugDataDumper.saveInvalidBlobSidecars(
-            blobSidecarsAndValidationResult.getBlobSidecars(), block);
+        LOG.error("sidecars validation result: {}", dataAndValidationResult::toLogString);
```

Comment:
@zilm13 do we want to keep this as ERROR?

Reply:
debug I think